app/Http/Transformers/ReportbackTransformer.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Rogue\Http\Transformers;
+
+use Rogue\Models\Reportback;
+use League\Fractal\TransformerAbstract;
+
+class ReportbackTransformer extends TransformerAbstract
+{
+    /**
+     * Transform resource data.
+     *
+     * @param User $user
+     * @return array
+     */
+    public function transform(Reportback $reportback)
+    {
+        return [
+            'id' => (string) $reportback->id,
```

Comment:
it comes back as a string. I was just trying to be as explicit as possible in the transformer, but doesn't seem necessary